packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -179,7 +180,20 @@ class SwitchListTile extends StatelessWidget {
     this.onFocusChange,
     this.enableFeedback,
     this.hoverColor,
+    this.onActiveThumbImageError,
```

Comment:
Can you please reorder the member definitions and constructor params to match `Switch`? Otherwise, it'll be easy to get lost in the future

Reply:
Fixed!